tomopy/prep/alignment.py

```diff
@@ -443,3 +455,240 @@ def shift_images(prj, sx, sy):
     prj *= scl
 
     return prj
+
+
+def find_slits_corners_aps_1id(img, 
+                               method='quadrant+', 
+                               medfilt2_kernel_size=3,
+                               medfilt_kernel_size=23,
+                               ):
+    """
+    Automatically locate the slit box location by its four corners.
+
+    Parameters
+    ----------
+    img                   :  np.2darray
+        input 2D images
+    method                :  str,  ['simple', 'quadrant', 'quadrant+'], optional

```

Comment:
How do simple, quadrant, quadrant+ differ from each other? Some explanation for the user is needed.